generators/php/codegen/src/context/PhpAttributeMapper.ts

```diff
@@ -71,12 +74,16 @@ export class PhpAttributeMapper {
         });
     }
 
-    public getUnionTypeParameters(types: php.Type[]): php.AstNode[] {
+    public getUnionTypeParameters({
+        types,
+        isOptional = false
```

Comment:
I think we can drop this since the fact that it's typed as `isOptional?` the value will be `false`.